task/buildah-oci-ta/0.1/buildah-oci-ta.yaml

```diff
@@ -183,7 +193,7 @@ spec:
       sed -i 's/^\s*short-name-mode\s*=\s*.*/short-name-mode = "disabled"/' /etc/containers/registries.conf
 
       # Setting new namespace to run buildah - 2^32-2
-      echo 'root:1:4294967294' | tee -a /etc/subuid >> /etc/subgid
+      echo 'root:1:4294967294' | tee -a /etc/subuid >>/etc/subgid
```

Comment:
NIT: I do prefer the whitespace, but it's not a big deal if it's much of an effort to keep those in.

Reply:
Done by the shell parser/formatter and the https://pkg.go.dev/mvdan.cc/sh/v3@v3.8.0/syntax#SpaceRedirects doesn't help it seems